terraform/environments/digital-prison-reporting/temporary_domain_refresh_jobs.tf

```diff
@@ -0,0 +1,282 @@
+  command_type                  = "glueetl"
+  description                   = "Monitors the reporting hub for table changes and applies them to domains"
+  create_security_configuration = local.create_sec_conf
+  job_language                  = "scala"
+  temp_dir                      = "s3://${module.s3_glue_job_bucket.bucket_id}/tmp/${local.domain_refresh_prisoner_prisoner}/"
+  checkpoint_dir                = "s3://${module.s3_glue_job_bucket.bucket_id}/checkpoint/${local.domain_refresh_prisoner_prisoner}/"
+  spark_event_logs              = "s3://${module.s3_glue_job_bucket.bucket_id}/spark-logs/${local.domain_refresh_prisoner_prisoner}/"
+  # Placeholder Script Location
+  script_location              = local.glue_placeholder_script_location
+  enable_continuous_log_filter = false
+  project_id                   = local.project
+  aws_kms_key                  = local.s3_kms_arn
+  additional_policies          = module.kinesis_stream_ingestor.kinesis_stream_iam_policy_admin_arn
+  # timeout                       = 1440
+  execution_class             = "FLEX"
+  worker_type                 = local.refresh_job_worker_type
+  number_of_workers           = local.refresh_job_num_workers
+  max_concurrent              = 64
+  region                      = local.account_region
+  account                     = local.account_id
+  log_group_retention_in_days = 1
+
+  tags = merge(
+    local.all_tags,
+    {
+      Name          = "${local.domain_refresh_prisoner_prisoner}"
+      Resource_Type = "Glue Job"
+      Jira          = "DPR2-193"
+    }
+  )
+
+  arguments = {
+    "--extra-jars"                   = local.glue_jobs_latest_jar_location
+    "--class"                        = "uk.gov.justice.digital.job.DomainRefreshJob"
+    "--datalake-formats"             = "delta"
+    "--dpr.aws.dynamodb.endpointUrl" = "https://dynamodb.${local.account_region}.amazonaws.com"
+    "--dpr.aws.kinesis.endpointUrl"  = "https://kinesis.${local.account_region}.amazonaws.com"
+    "--dpr.aws.region"               = local.account_region
+    "--dpr.curated.s3.path"          = "s3://${module.s3_curated_bucket.bucket_id}"
+    "--dpr.domain.registry"          = "${local.project}-domain-registry-${local.environment}"
+    "--dpr.domain.target.path"       = "s3://${module.s3_domain_bucket.bucket_id}"
+    "--dpr.domain.catalog.db"        = module.glue_data_domain_database.db_name
+    "--dpr.log.level"                = local.refresh_job_log_level
+    "--dpr.domain.name"              = "prisoner"
+    "--dpr.domain.table.name"        = "prisoner"
+    "--dpr.domain.operation"         = "sync"
+  }
+}
+
+# =======================================
+# SCHEDULES
+# =======================================
+
+# Establishment Establishment Schedule
+resource "aws_glue_trigger" "temp_domain_refresh_establishment_establishment" {
+  name     = "${local.domain_refresh_establishment_establishment}-trigger"
+  schedule = "cron(*/15 06-20 * * MON-FRI)"
```

Comment:
I think AWS cron expressions have 6 elements rather than the usual 5 you need with standard crontabs. So I think you'd want an extra "*" at the end for year? https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-cron-expressions.html